src/node/rpc/common_endpoint_registry.h

```diff
@@ -247,7 +249,7 @@ namespace ccf
 
         for (auto& [verb, endpoint] : it->second)
         {
-          std::string verb_name = http_method_str(verb);
+          std::string verb_name = verb.c_str();
```

Comment:
is this right? I am assuming that verb is a std::string?

Reply:
Verb is an RPCVerb now.

Reply:
Here we allocate a copy because we lowercase it for the schema.